sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+  some PVC-specific flags even if target device is not a PVC. intel/llvm#13794
+- Fixed a bug with incorrect file extensions being emitted in AOT compilation
+  when `--save-temps` is used. intel/llvm#14214
+- Made `-fsycl-add-default-spec-consts-image` available with `clang-cl`. intel/llvm#13168
+- Fixed an issue where performing separate compilation and linking with
+  `-fsycl-link` would result in "number of output files and targets should match
+  in unbundling mode" error emitted by the compiler during link step. intel/llvm#13002
+- Fixed a bug in address sanitizer which may lead to crashes when an application
+  is launched on OpenCL CPU device. intel/llvm#13262
+- Fixed a bug where calling certain built-in math functions that accepts
+  pointers (like `fract`, `frexp`, `modf`, etc.) and passing pointer in the
+  generic address space there would not compile for AMD devices.
+  intel/llvm#13015 intel/llvm#13092 intel/llvm#13361 intel/llvm#13792
+  intel/llvm#13546
+- Fixed a bug where compiling a program that contains kernels with different
+  `reqd_work_group_size` attributes attached to them using
+  `-fsycl-device-code-split=none` would result in an exception being thrown at
+  runtime about mismatching work-group size. intel/llvm#13523
+- Fixed a bug where compiling a kernel that is annotated with
+  `reqd_work_group_size` attribute that has less than 3 arguments for HIP
+  target caused compiler to crash. intel/llvm#13600
+- Fixed a bug with `shift_group_[right|left]`, `permute_by_xor` and
+  `select_from_group` algorithms would return invalid values if used with
+  `half` data type on AMD devices. intel/llvm#13016
+
+### SYCL Library
+
+- Fixed a situation when querying
+  `sycl::ext::oneapi::experimental::info::device` could result in exception
+  being thrown instead of empty vector being returned. intel/llvm#13968
+- Fixed `esimd::atan` implementation under `-ffast-math` flag. intel/llvm#13186
+- Fixed an issue that component devices were not considered to be a descendent
+  from their composite devices when creating a queue. intel/llvm#13513
+- Fixed an issue that querying for
+  `ext::oneapi::experimental::info::device::composite_device` would *not*
+  throw an exception if device is not a component device. intel/llvm#13868
+- Fixed an issue that querying for composite devices may result in some devices
+  returned twice. intel/llvm#14442
+- Fixed a bug that querying for composite devices may result in an exception
+  being thrown instead of an empty vector being returned. intel/llvm#13931
+- Fixed a bug in copy-constructor of `config_2d_mem_access` ESIMD class which
+  would lead to compilation errors. intel/llvm#13632
+- Fixed an issue that use of `atomic_ref<T *>` would not be detected as a use
+  of `atomic64` aspect leading to errors due to speculative compilation. intel/llvm#14052
+- Fixed `ctanh` and `cexp` returning incorrect value in some edge cases. intel/llvm#14329
```

Comment:
```suggestion
- Fixed `ctanh` and `cexp` returning incorrect values in some edge cases. intel/llvm#14329
```

Reply:
Applied in 1fe2c554292d3a9d90f952c02ff4b8e3dfaab706